build/linux/Makefile

```diff
@@ -1,5 +1,5 @@
 TOP:=../..
-CXX?=clang++-3.9
+CXX=clang++-3.8
```

Comment:
The ?= means that we can set CXX as a command line argument for the make process... maybe that's preferable to trying to get one value that works everywhere.